### PR TITLE
SAK-32371 : Resources doesnt use font awesome icons

### PR DESCRIPTION
**Pull request by frasese**
More comments in JIRA : https://jira.sakaiproject.org/browse/SAK-32371

### CODE REVIEW

**Comment by mpellicer** [comment]
Wow @frasese,  amazing work!

File: content/content-tool/tool/src/webapp/vm/content/chef_resources_deleteConfirm.vm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 					<tr>
 						<td headers="title">
 							<h4>
-								<img src = "#imageLink($member.iconLocation)" alt ="$member.label" />
+								<span class="$member.iconClass" title="$!member.label"></span>
```
Comment (master-bob) [inline content/content-tool/tool/src/webapp/vm/content/chef_resources_deleteConfirm.vm]: Good work using `span`!

The alt tag is especially useful for accessibility purposes. The W3C does not recommend using the title attribute for accessibility purposes, [source](https://www.w3.org/TR/html/dom.html#the-title-attribute). I *think* the icon just provides visual styling, in which case I would add the `aria-hidden="true"` attribute.

If the icon serves a semantic purpose than the following changes should be made:
- Add <span class="sr-only">$!member.label</span> after the icon.

Adding the `sr-only span`, would correctly capture what was used previously.

**Comment by master-bob** [comment]
Please make sure that accessibility is maintained throughout your improvements.

File: content/content-tool/tool/src/webapp/vm/content/chef_resources_deleteConfirm.vm (hunk 1 above)
Comment (ottenhoff) [inline content/content-tool/tool/src/webapp/vm/content/chef_resources_deleteConfirm.vm]: alt isn't relevant for span tag. I agree re aria-hidden="true"

**Comment by frasese** [comment]
Rebased

**Comment by ern** [comment]
Just a quick thought on this does it make sense that these images are setup in a properties file?

Would this be better implemented as a skin asset which can be configured there. Meaning create a namespace very similar to how icons in sakai are done for content types. Then this becomes a task related to skinning sakai which is germane to morpheus?

**Comment by mpellicer** [comment]
Hi @ern, thanks for the suggestions., we've just respected the current implementation, i agree is better to implement it in that way but requires a big effort and we have an important number of open cases.

Do you think this can be merged with the current implementation?

Regards

**Comment by ern** [comment]
I understand priorities so yeah it can be merged with the current implementation, but maybe could make a JIRA expressing indicating this could be better implemented in the skin ;)

**Comment by mpellicer** [comment]
Agree @ern, also some tools will need an update to move img icons to fa icons

**Pull request merged**